stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChatUI.kt

```diff
@@ -0,0 +1,44 @@
+package io.getstream.chat.android.ui
+
+import android.content.Context
+import com.getstream.sdk.chat.style.ChatFonts
+import com.getstream.sdk.chat.style.ChatStyle
+import io.getstream.chat.android.ui.common.UrlSigner
+import io.getstream.chat.android.ui.common.markdown.ChatMarkdown
+import io.getstream.chat.android.ui.common.markdown.ChatMarkdownImpl
+import io.getstream.chat.android.ui.common.style.ChatFontsImpl
+
+/**
+ * ChatUI handles any configuration for the Chat UI elements. It replaces the older Chat class.
```

Comment:
Probably don't need to mention this anymore

```suggestion
 * ChatUI handles any configuration for the Chat UI elements.
```